src/main/java/frc/robot/maps/PracticeBot.java

```diff
@@ -47,6 +47,11 @@ public SparkMaxCounter getHeightEncoder() {
             public DoubleSolenoid getBrake() {
                 return new DoubleSolenoid(2, 7);
             }
+
+            @Override
+            public DoubleSolenoid getArmsPiston() {
+                return null;
```

Comment:
This should return a DoubleSolenoid so the lift doesn't break.

Reply:
> This should return a DoubleSolenoid so the lift doesn't break.

Fixed!